redbeams/src/test/java/com/sequenceiq/redbeams/configuration/DatabaseServerSslCertificateConfigIntegrationTest.java

```diff
@@ -185,6 +187,7 @@ void getSupportedPlatformsForCertsTest() {
                 Set.of(
                     "aws",
                     "aws." + REGION_EUS1,
```

Comment:
Please also update all the other test cases in the class, adding verifications for the new region. You will have to add a constants `CERT_ISSUER_*` and `CLOUD_PROVIDER_IDENTIFIER_*` with the right expected values.